### PR TITLE
GUI 

### PR DESCRIPTION
**Pull request by Jasmine-Zhuang**
Draft for GUI

### CODE REVIEW

**Comment by RyanF-DevPage** [comment]
Nice start!

**Comment by Jasmine-Zhuang** [comment]
The icon image is pushed to Flight branch, for the newest version, please see in Flight branch.

File: src/main/java/GUI.java [hunk 1]
```diff
@@ -1,14 +1,49 @@
+package GUI;
+
 import javax.swing.*;
+import javax.swing.border.Border;
```
Comment (oliviaj0422) [inline src/main/java/GUI.java:3]: line 3 already imports everything from swing, so we probably don't need lines 4 and 5

**Review by oliviaj0422 — COMMENTED** [review]
We still need to create some buttons and add them to the panel. This is probably our next step. Thanks for making this template.

File: src/main/java/GUI.java [hunk 2]
```diff
@@ -7,41 +7,53 @@
 
 public class GUI {
     public static void main(String[] args){
-        //frame setup
-        JFrame frame = new JFrame("UTicket");
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.setSize(300,300);
```
Comment (oliviaj0422) [inline src/main/java/GUI.java]: feel like the size could be bigger, like 500 by 500 or something

File: src/main/java/GUI.java (hunk 1 above)
Comment (Jasmine-Zhuang) [reply src/main/java/GUI.java:3]: Not sure, but Inteliij warns me to do so.

**Pull request closed without merging**